packages/lit-starter-ts/package.json

```diff
@@ -41,37 +41,30 @@
   "author": "Google LLC",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "lit": "^2.1.0"
+    "lit": "^2.2.4"
   },
   "devDependencies": {
-    "@11ty/eleventy": "^0.12.1",
-    "@11ty/eleventy-plugin-syntaxhighlight": "^3.0.1",
+    "@11ty/eleventy": "^1.0.1",
+    "@11ty/eleventy-plugin-syntaxhighlight": "^4.0.0",
     "@custom-elements-manifest/analyzer": "^0.5.7",
-    "@esm-bundle/chai": "^4.1.5",
-    "@open-wc/testing": "^3.0.0-next.1",
-    "@open-wc/testing-karma": "^4.0.9",
-    "@rollup/plugin-node-resolve": "^13.0.4",
-    "@rollup/plugin-replace": "^3.0.0",
-    "@types/mocha": "^9.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.3.0",
-    "@typescript-eslint/parser": "^4.30.0",
-    "@web/dev-server": "^0.1.22",
-    "@web/dev-server-legacy": "^0.1.4",
-    "@web/dev-server-rollup": "^0.3.9",
-    "@web/test-runner": "^0.13.16",
-    "@web/test-runner-mocha": "^0.7.4",
-    "@web/test-runner-playwright": "^0.8.4",
+    "@open-wc/testing": "^3.1.5",
+    "@rollup/plugin-node-resolve": "^13.3.0",
+    "@rollup/plugin-replace": "^4.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.25.0",
+    "@typescript-eslint/parser": "^5.25.0",
+    "@web/dev-server": "^0.1.31",
+    "@web/dev-server-legacy": "^1.0.0",
+    "@web/test-runner": "^0.13.27",
+    "@web/test-runner-playwright": "^0.8.8",
     "@webcomponents/webcomponentsjs": "^2.6.0",
-    "deepmerge": "^4.2.2",
-    "eslint": "^7.32.0",
-    "lit-analyzer": "^1.1.10",
-    "mocha": "^9.1.1",
-    "prettier": "^2.3.2",
+    "eslint": "^8.15.0",
+    "lit-analyzer": "^1.2.1",
+    "prettier": "^2.6.2",
     "rimraf": "^3.0.2",
-    "rollup": "^2.70.2",
-    "rollup-plugin-summary": "^1.2.3",
+    "rollup": "^2.73.0",
+    "rollup-plugin-summary": "^1.4.3",
     "rollup-plugin-terser": "^7.0.2",
-    "typescript": "^4.3.5"
+    "typescript": "^4.6.4"
```

Comment:
Let's change this to a `~` because TypeScript makes breaking changes on minor releases.

Reply:
Done!